### PR TITLE
Bugfix - Add validation to stop_date field in project

### PR DESCRIPTION
**Pull request by Szymiks**
Resolves: https://github.com/Code-Poets/sheetstorm/issues/241

I think that removed test is useless because in opinion it is already tested by django, but you guys could say more about it :) Thanks!

### CODE REVIEW

File: managers/models.py [hunk 1]
```diff
@@ -45,6 +47,12 @@ def __str__(self) -> str:
     def get_report_ordered(self) -> QuerySet:
         return self.report_set.select_related("task_activities").order_by("author__email", "-date", "-creation_date")
 
+    def clean(self) -> None:
+        super().clean()
+        if self.stop_date is not None:
+            if self.start_date > self.stop_date:
```
Comment (rwrzesien) [inline managers/models.py]: This should be one-line condition.

File: managers/tests/test_unit_model.py [hunk 2]
```diff
@@ -99,3 +96,11 @@ def test_project_model_members_field_should_accept_correct_input(self):
         project.save()
         project.members.add(self.member)
         self.assertTrue(project.members.all().filter(email=self.member.email).exists())
+
+    def test_project_model_end_date_could_not_be_before_start_day(self):
```
Comment (rwrzesien) [inline managers/tests/test_unit_model.py]: `could_not_be` => `should_not_be`

File: managers/tests/test_unit_model.py [hunk 3]
```diff
@@ -99,3 +96,11 @@ def test_project_model_members_field_should_accept_correct_input(self):
...
+        project = self.default_model()
+        project.stop_date = project.start_date - timedelta(1)
+        with self.assertRaises(ValidationError):
+            project.full_clean()
+            project.save()
+        self.assertTrue(not Project.objects.all().exists())
```
Comment (rwrzesien) [inline managers/tests/test_unit_model.py]: Use `self.assertFalse()`

File: managers/models.py (hunk 1 above)
Comment (Szymiks) [reply managers/models.py]: Done

File: managers/tests/test_unit_model.py (hunk 2 above)
Comment (Szymiks) [reply managers/tests/test_unit_model.py]: lol what I wrote.. :dagger: Done

File: managers/tests/test_unit_model.py (hunk 3 above)
Comment (Szymiks) [reply managers/tests/test_unit_model.py]: Done

**Pull request merged**